cmd/protoc-gen-go-grpc/grpc.go

```diff
@@ -374,7 +409,8 @@ func genServerMethod(gen *protogen.Plugin, file *protogen.File, g *protogen.Gene
 		g.P("if interceptor == nil { return srv.(", service.GoName, "Server).", method.GoName, "(ctx, in) }")
 		g.P("info := &", grpcPackage.Ident("UnaryServerInfo"), "{")
 		g.P("Server: srv,")
-		g.P("FullMethod: ", strconv.Quote(fmt.Sprintf("/%s/%s", service.Desc.FullName(), method.Desc.Name())), ",")
+		serviceName := helper.formatServiceName(service)
```

Comment:
Hmm, what if the method is also renamed? Is that never changed?
Maybe make this take service and method, and return full name.